lib/publiccloud/gcp_client.pm

```diff
@@ -35,7 +35,7 @@ sub init {
     assert_script_run('source ~/.bashrc');
     if (is_sle('>=15')) {
         # kill it in case it's running
-        script_run("killall chronyd && sleep 5 && if pgrep chronyd; then killall -9 chronyd; fi");
+        assert_script_run('systemctl stop chronyd.service && sleep 5 && if pgrep chronyd; then kill -9 chronyd; fi');
```

Comment:
kill command does not support string but requires pid . only killall accepting strings